accounts/forms.py

```diff
@@ -0,0 +1,11 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm, UsernameField
+
+User = get_user_model()
+ 
+class CustomUserCreationForm(UserCreationForm):
+    class Meta:
+        model = User
+        fields = ("username",)
+        fields_classes = {"username": UsernameField}
```

Comment:
Other places use single quotes. Maintain consistency in your entire codebase. Either use singlequotes everywhere or double quotes everywhere